editing/edit-features-with-feature-linked-annotation/README.metadata.json

```
@@ -0,0 +1,34 @@
+{
+    "category": "",
```

Let's not forget to add the category here. 
If you haven't been using the script to generate this, do give it a shot. We're constantly optimising it so it would be helpful to see the output with every new sample
```suggestion
    "category": "Editing",
```